stdlib/public/Concurrency/Clock.swift

```diff
@@ -67,7 +67,18 @@ extension Clock {
   ///          await someWork()
   ///       }
   @available(SwiftStdlib 5.7, *)
+  @_alwaysEmitIntoClient
   public func measure(
+    isolation: isolated (any Actor)? = #isolation,
```

Comment:
This change will put `public func measure` behind `$OptionalIsolatedParameters` in the swiftinterface for the _Concurrency library, meaning the function will disappear when imported with a compiler that predates `OptionalIsolatedParameters`. I need to fix this before merging this change.

Reply:
Fixed using `CONDITIONALLY_SUPPRESSIBLE_LANGUAGE_FEATURE`:

```
  #if compiler(>=5.3) && $ExpressionMacroDefaultArguments
  #if $OptionalIsolatedParameters
  @available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
  @_alwaysEmitIntoClient public func measure(isolation: isolated (any _Concurrency.Actor)? = #isolation, _ work: () async throws -> Swift.Void) async rethrows -> Self.Instant.Duration {
    try await measure(work)
  }
  #else
  @available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
  @_alwaysEmitIntoClient public func measure(isolation: (any _Concurrency.Actor)? = nil, _ work: () async throws -> Swift.Void) async rethrows -> Self.Instant.Duration {
    try await measure(work)
  }
  #endif
  #endif
```